components/brave_sync/brave_profile_sync_service_impl.h

```diff
@@ -215,6 +215,16 @@ class BraveProfileSyncServiceImpl
 
   bool IsSQSReady() const;
 
+  // Methods to run specified actions right after bookmark model was loaded
+  void RunWhenModelIsLoaded(base::OnceCallback<void()> fn,
+                            const std::string& fn_name);
+  void CtorBkmLoaded(Profile* profile);
```

Comment:
please don't abbreviate things like this in method names. We should use the same terminology everywhere for consistency

Reply:
thanks, now this is fixed